dbms/src/AggregateFunctions/AggregateFunctionMerge.h

```diff
@@ -64,6 +64,16 @@ class AggregateFunctionMerge final : public IAggregateFunctionHelper<AggregateFu
         nested_func->merge(place, static_cast<const ColumnAggregateFunction &>(*columns[0]).getData()[row_num], arena);
     }
 
+    void decrease(AggregateDataPtr __restrict place, const IColumn ** columns, size_t row_num, Arena * arena)
```

Comment:
`decrease` is the same as `add`?

Reply:
> `decrease` is the same as `add`?

`AggregateFunctionMerge` is impossible to be used in window agg. It's just a placeholder to pass the compilation. I will delete it.